packages/nc-gui-v2/layouts/base.vue

```diff
@@ -43,6 +47,22 @@ const logout = () => {
 
           <div class="flex-1" />
 
+          <div class="flex items-center mr-4">
+            <a-button
+              v-if="isUIAllowed('newUser')"
+              size="middle"
+              type="primary"
+              class="!bg-white !text-primary rounded"
+              @click="showUserModal = true"
+            >
+              <div class="flex flex-row justify-center items-center caption capitalize space-x-1">
```

Comment:
Is caption a vuetify class? Will not exist soon

Reply:
Good catch. will remove it.